spec/policies/kudo_policy_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+require 'pundit/rspec'
+
+describe KudoPolicy do
+  include ActiveSupport::Testing::TimeHelpers
```

Comment:
You could move it to `rails_helper.rb` or `spec_helper.rb` instead of putting it in each spec where you need it.

Reply:
I know, but I do not want to load it in every test. They take enough time on CI already.